packages/ui/atoms/CircularMarquee/CircularMarquee.ts

```diff
@@ -0,0 +1,63 @@
+import { Base, BaseConfig, BaseProps } from '@studiometa/js-toolkit';
+import { damp, transform } from '@studiometa/js-toolkit/utils';
+
+export interface CircularMarqueeProps extends BaseProps {
+  $options: {
+    sensitivity: number;
+  };
+  rotate: number;
+  deltaY: number;
+  transform: object;
```

Comment:
**chore :** Remove these properties

Instance properties declared on the class does not need to be repeated here. The `...Props` interface is only necessary for properties from the framework API: `$options`, `$el`, `$refs` and `$children`. 

```suggestion
```

Reply:
👍  add doubt about if they should be here or not when writing it